### PR TITLE
Side 2425 component tooltip

### PR DESCRIPTION
**Pull request by RusGraf**
Creating tooltips for button, markdown and message controls.

### CODE REVIEW

File: ldk/javascript/examples/self-test-loop/src/selfTest.ts [hunk 1]
```diff
@@ -44,6 +35,18 @@ import {
   fileExists,
 } from './tests';
 
+import {
+  testBoxInTheBox,
```
Comment (cshields-oliveai) [inline ldk/javascript/examples/self-test-loop/src/selfTest.ts:39]: :D

File: ldk/javascript/examples/self-test-loop/src/tests/whisper.ts [hunk 2]
```diff
@@ -0,0 +1,700 @@
...
+                  form.close((error) => console.log(error));
+                  reject(false);
+                },
+              },
+            ],
+          },
+        ],
+      });
+    } catch (error) {
+      console.error(error);
```
Comment (cshields-oliveai) [inline ldk/javascript/examples/self-test-loop/src/tests/whisper.ts:307]: Might want to close the whispers as well upon rejection.

File: ldk/javascript/examples/self-test-loop/src/tests/whisper.ts [hunk 3]
```diff
@@ -0,0 +1,700 @@
+import { clipboard, whisper } from '@oliveai/ldk';
```
Comment (cshields-oliveai) [inline ldk/javascript/examples/self-test-loop/src/tests/whisper.ts:1]: Is this diff not showing these tests moved from `self-test-loop/src/tests/index.ts` or were they not removed?

**Review by cshields-oliveai — APPROVED** [review]

File: ldk/javascript/examples/self-test-loop/src/tests/whisper.ts (hunk 2 above)
Comment (cshields-oliveai) [reply ldk/javascript/examples/self-test-loop/src/tests/whisper.ts:312]: Even though I don't believe you added this.

Comment (RusGraf) [reply ldk/javascript/examples/self-test-loop/src/tests/whisper.ts:312]: I added a timeout form close to close the whisper if no action from the user received. Although we cant do it in the catch as it means that whisper is not properly created and there is no instance to close.

File: ldk/javascript/examples/self-test-loop/src/tests/whisper.ts (hunk 3 above)
Comment (RusGraf) [reply ldk/javascript/examples/self-test-loop/src/tests/whisper.ts:1]: Well, looks like merging from main brought them back. Let me fix it.

Comment (RusGraf) [reply ldk/javascript/examples/self-test-loop/src/tests/whisper.ts]: Cleaned up

**Review by epiccoleman — APPROVED** [review]

**Pull request merged**